### DISCUSSION TITLE
Can I build this with a Mac?

### DISCUSSION THREAD

**Discussion by swwol** [Zer0-bit/gaggiuino#203] [Q&A] [answered]
Interested in building this - but I don't have access to a windows machine. It looks like the Arduino driver and the Nextion editor are only available for windows. Will a Mac be able to program the Arduino - maybe I need to get a genuine one rather than a clone?

thanks for all your work on this looks amazing!

**Comment by Zer0-bit** [comment, marked as answer]
You can definitely build it using a mac, there's numerous people using macs that have done it, the arduino driver can be found for mac as well though if you're using the latest processors apparently not all mac compatible drivers work.
In regards to Nextion you don't really need to run their software, you just need the pre-compiled .tft file drop it on a sd card and upload to the lcd module.
Anyhow it's for sure 100% possible and no reasons to fear that it isn't.
If you have more questions might be sensible joining discord for some help from people who've been in your shoes and got the questions answered.
